app/models/notification.rb

```diff
@@ -0,0 +1,21 @@
+class Notification < ActiveRecord::Base
+  acts_as_readable on: :updated_at
+  belongs_to :user
+  belongs_to :course
+
+  scope :center_popup, -> { where(type: 'CenterPopup') }
+  scope :right_side_popup, -> { where(type: 'RightSidePopup') }
+  scope :email, -> { where(type: 'EmailNotification') }
+
+  def self.types
```

Comment:
why we need a function here ? 
http://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html